### PR TITLE
Fix/connection map colors

### PR DESCRIPTION
**Pull request by goodpickle**
## Description

This PR fixes the ConnectionMap colors as well as updating the build tests to use snapshots instead of needing manual update.

### Before

Screen Shot 2022-09-28 at 2 22 29 PM

### After

Screen Shot 2022-09-28 at 2 22 04 PM

## Todo

- [ ] Bump version and add tag

### CODE REVIEW

**Comment by evad1n** [comment]
It looks like this fills in the country with blue now? The vercel preview gets blocked by CORS, so what do you think about including a screenshot to make it easier to visualize this change and see the difference?

**Comment by goodpickle** [comment]
> It looks like this fills in the country with blue now? The vercel preview gets blocked by CORS, so what do you think about including a screenshot to make it easier to visualize this change and see the difference?

@evad1n The description has been updated.

**Review by evad1n — DISMISSED** [review]
Looks great! Appreciate the pictures~

File: build-utils/css/README.md [hunk 1]
```diff
@@ -1,149 +1,158 @@
 # **Build styles**
-This script is primarily created for generating accessible style constants and css file from configured style constant to benefit from strongly typed css variable and jsdocs support for Amino consumers.   
-  
+
+This script is primarily created for generating accessible style constants and css file from configured style constant to benefit from strongly typed css variable and jsdocs support for Amino consumers.
```
Comment (Thienhuynh95) [inline build-utils/css/README.md:3]: Thanks for organizing the README.

**Review by Thienhuynh95 — APPROVED** [review]
Awesome refactor! Thanks

**Review by ckgardner — COMMENTED** [review]
Love the snapshot usage! I'm excited to dive into that.

**Pull request merged**